src/server/db/audit-report.js

```diff
@@ -8,10 +8,10 @@
 */
 const knex = require('./connection')
 
-let log = () => {}
-if (process.env.VERBOSE) {
-  log = console.log
-}
+// let log = () => {}
```

Comment:
what's the goal with commenting these lines out vs. just removing them?

Reply:
I don't have a strong opinion here, happy to just delete it.  I don't really know what this log helper is for.

I guess I was just anticipating that it may still be useful for debugging in the future.  If I delete it, I was just worried we would lose the context that this env variable is available to control logging levels.

Now that we've had this conversation, maybe that's enough that we'll both remember?  🤷